rails/app/controllers/api/v1/report_users_controller.rb

```diff
@@ -50,6 +50,11 @@ def external_report_query
       type: "users",
       version: "1.0",
       domain: URI.parse(APP_CONFIG[:site_url]).host,
+      user: {
+        id: url_for(current_user),
```

Comment:
This might also suffer the same learn-report.concord.org problem

Reply:
The current AWS learner report gets the domain from the `learnersApiUrl` attribute which has `-report` in it and that it what is used when redirecting to the portal report "watcher".

Reply:
Example:

`https://researcher-reports.concord.org/branch/master/?portal=https%3A%2F%2Flearn-report.staging.concord.org`

Reply:
That is why I didn't use the `APP_CONFIG[:site_url]` value like is used in line 52.